doc/conf.py

```diff
@@ -16,24 +16,68 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
+
+# pylint: skip-file
+
 import os
 import sys
-sys.path.insert(0, os.path.abspath('..'))
 
+sys.path.insert(0, os.path.abspath('../'))
+
+autodoc_mock_imports = ['deepspeech']
+
+read_the_docs_build = os.environ.get('READTHEDOCS', None) == 'True'
+if read_the_docs_build:
+    import subprocess
+    subprocess.call('cd ../ && doxygen doc/doxygen-c.conf', shell=True)
+    subprocess.call('cd ../ && doxygen doc/doxygen-java.conf', shell=True)
 
 # -- General configuration ------------------------------------------------
 
+import semver
+
+# -- Project information -----------------------------------------------------
+
+project = 'deepspeech-api'
+copyright = '2019, DeepSpeech Team'
```

Comment:
nit: Mozilla Corporation?

Reply:
copyright = MoCo ?

Reply:
That's my understanding of our work contract, yes :P

Reply:
Dunno if France has different rules.